DfE.FindInformationAcademiesTrusts/Pages/Cookies.cshtml

```diff
@@ -2,139 +2,151 @@
+                    <td class="govuk-table__cell">With browser settings (Session)</td>
+                </tr>
+            </tbody>
+        </table>
 
-    <h2 class="govuk-heading-m">Optional analytics cookies</h2>
-    <p class="govuk-body">
-      With your permission, we use Application Insights to collect data about how you use this site. This information helps us to improve our service.
-    </p>
-    <p class="govuk-body">
-      Application Insights is not allowed to use or share our analytics data with anyone. Application Insights stores anonymised information about:
-    </p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>how you got to this site</li>
-      <li>the pages you visit on this site and how long you spend on them</li>
-      <li>any errors you see while using this site</li>
-      <li>what you click on while you are visiting the site</li>
-    </ul>
+        <h2 class="govuk-heading-m">Optional analytics cookies</h2>
+        <p class="govuk-body">
+            With your permission, we use Application Insights and Google Analytics to collect data about how you use this site. This information helps us to improve our service.
+        </p>
+        <p class="govuk-body">
+            Application Insights and Google Analytics are not allowed to use or share our analytics data with anyone. Application Insights and Google Analytics store information about:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>how you got to this site</li>
+            <li>the pages you visit on this site and how long you spend on them</li>
+            <li>any errors you see while using this site</li>
+            <li>what you click on while you are visiting the site</li>
+            <li>whether you’ve visited before</li>
+            <li>your unique user identity</li>
+        </ul>
 
-    <table class="govuk-table" aria-label="Essential cookies">
-      <thead class="govuk-table__header">
-        <tr>
-          <th class="govuk-table__header">Name</th>
-          <th class="govuk-table__header">Purpose</th>
-          <th class="govuk-table__header">Expires</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <tr>
-          <td class="govuk-table__cell">ai_user</td>
-          <td class="govuk-table__cell">Checks if you have visited this site before. This helps us count how many people visit our site.</td>
-          <td class="govuk-table__cell">1 year</td>
-        </tr>
-        <tr>
-          <td class="govuk-table__cell">ai_session</td>
-          <td class="govuk-table__cell">Checks if you have visited this site before. This helps us count how many people visit our site.</td>
-          <td class="govuk-table__cell">30 minutes</td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
+        <table class="govuk-table" aria-label="Essential cookies">
```

Comment:
Can you change the spacing in this file and the other razor pages back to 2 spaces please?